sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_common/_response_handlers.py

```diff
@@ -29,21 +29,24 @@
 def _parse_schema_properties_dict(response):
     return {
         'id': response.headers.get('schema-id'),
-        'format': response.headers.get('serialization-type'),
         'version': int(response.headers.get('schema-version'))
     }
 
 
-def _parse_response_schema_properties(response):
+def _parse_response_schema_properties(response, format):
     properties_dict = _parse_schema_properties_dict(response)
-    properties_dict['id'] = response.json()["id"]
+    properties_dict['id'] = response.headers.get("schema-id")
```

Comment:
It seems like we don't need it here since `_parse_schema_properties_dict` is already trying to get("schema-id") from header
```suggestion
```